### PR TITLE
Orders Done

### PR DESCRIPTION
**Pull request by LogansZeroes**
specs and stuff passing

### CODE REVIEW

File: server/app/routes/orders/index.js [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+
+var Order = require('mongoose').model('Order');
+
+router.get('/', function (req, res) {
+    var query = {};
+    if (req.query.status) {
+        query.status = req.query.status;
+    }
+	Order.find(query)
+	.then(function (users) {
```
Comment (scolacur) [inline server/app/routes/orders/index.js]: this should probably say orders just to be clearer

**Comment by seanjohnite** [comment]
I committed again to this one if you guys didn't notice--so if you update it again, be sure to pull first. I changed the product to actually use really product ids :)

File: server/app/routes/orders/index.js [hunk 2]
```diff
@@ -0,0 +1,52 @@
+'use strict';
```
Comment (seanjohnite) [inline server/app/routes/orders/index.js:1]: This file needs error handling set up

File: server/app/routes/orders/orders-checkout.js [hunk 3]
```diff
@@ -0,0 +1,13 @@
+'use strict';
```
Comment (seanjohnite) [inline server/app/routes/orders/orders-checkout.js:1]: did you guys require this file anywhere? it doesn't look like it's wired up.

File: server/db/models/order.js [hunk 4]
```diff
@@ -0,0 +1,82 @@
...
+schema.plugin(deepPopulate);
+
+// schema.path('user').validate(function (value) {
+//     return !this.session;
+// }, 'session stuff');
+//
+// schema.path('session').validate(function (value) {
+//     return !this.user;
+// });
+
```
Comment (seanjohnite) [inline server/db/models/order.js:54]: What if it has a session and a user property? I think by the spec it should have either one or the other.. that's why I put these in there.

File: server/app/routes/orders/orders-checkout.js (hunk 3 above)
Comment (dwschrashun) [inline server/app/routes/orders/orders-checkout.js:1]: nope, that's a todo

File: server/db/models/order.js (hunk 4 above)
Comment (dwschrashun) [inline server/db/models/order.js:54]: addressed by making session required, and user not required

**Pull request merged**